cnf-tests/testsuites/pkg/namespaces/namespaces.go

```diff
@@ -131,6 +131,24 @@ func Create(namespace string, cs corev1client.NamespacesGetter) error {
 	return err
 }
 
+func AddPSALabelsToNamespace(namespace string, cs corev1client.NamespacesGetter) error {
+	ns, err := cs.Namespaces().Get(context.Background(), namespace, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	if ns.Labels == nil {
+		ns.Labels = GetPSALabels()
+		return nil
+	}
+
+	for k, v := range GetPSALabels() {
+		ns.Labels[k] = v
+	}
+
+	return nil
```

Comment:
This function has no effect. Maybe it is missing an Update on cluster?

```suggestion
    _, err = cs.Namespaces().Update(context.Background(), ns, metav1.UpdateOptions{})
    return err
```